static/js/StaticPages.jsx

```diff
@@ -2636,6 +2636,251 @@ const StaticHR = () =>
+    <>
+        {teamMembers.map((teamMember) => (
+            <TeamMember key={teamMember.id} teamMember={teamMember} />
+        ))}
+    </>
+);
+
+const BoardMember = ({ boardMember }) => (
+    <div className="teamBoardMember">
+        <TeamMemberDetails teamMemberDetails={boardMember.teamMemberDetails} />
+    </div>
+);
+
+const BoardMembers = ({ boardMembers }) => {
+    let chairmanBoardMember;
+    const chairmanIndex = boardMembers.findIndex(
+        (boardMember) =>
+            boardMember.teamMemberDetails.teamTitle.en.toLowerCase() ===
+            "chairman"
+    );
+    if (chairmanIndex !== -1) {
+        chairmanBoardMember = boardMembers.splice(chairmanIndex, 1);
+    }
+    const [cofounderBoardMembers, regularBoardMembers] = partition(
+        boardMembers,
+        (boardMember) =>
+            boardMember.teamMemberDetails.teamTitle.en.toLowerCase() ===
+            "co-founder"
+    );
+
+    return (
+        <>
+            {chairmanBoardMember && (
+                <BoardMember boardMember={chairmanBoardMember[0]} />
+            )}
+            {cofounderBoardMembers.map((boardMember) => (
+                <BoardMember key={boardMember.id} boardMember={boardMember} />
+            ))}
+            {regularBoardMembers.sort(byLastName()).map((boardMember) => (
+                <BoardMember key={boardMember.id} boardMember={boardMember} />
+            ))}
+        </>
+    );
+};
+
+const fetchTeamMembersJSON = async () => {
+    const query = `
+query {
+  teamMembers(pagination: { limit: -1 }) {
+    data {
+      id
+      attributes {
+        teamName
+        teamTitle
+        isTeamBoardMember
+        teamMemberImage {
+          data {
+            attributes {
+              url
+            }
+          }
+        }
+        localizations {
+          data {
+            attributes {
+              locale
+              teamName
+              teamTitle
+            }
+          }
+        }
+      }
+    }
+  }
+}
+`;
+    try {
+        const response = await fetch(STRAPI_INSTANCE + "/graphql", {
+            method: "POST",
+            mode: "cors",
+            cache: "no-cache",
+            credentials: "omit",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            redirect: "follow",
+            referrerPolicy: "no-referrer",
+            body: JSON.stringify({ query }),
+        });
+        if (!response.ok) {
+            throw new Error(`HTTP Error: ${response.statusText}`);
+        }
+        const data = await response.json();
+        return data;
+    } catch (error) {
+        throw error;
+    }
+};
+
+const TeamMembersPage = memo(() => {
+    const [ordinaryTeamMembers, setOrdinaryTeamMembers] = useState([]);
+    const [teamBoardMembers, setTeamBoardMembers] = useState([]);
+    const [error, setError] = useState(null);
+
+    useEffect(() => {
+        const loadTeamMembers = async () => {
```

Comment:
We discussed not defining functions where they are called. Plese move this function definition into the components scope and not the hook's scope